2019 updates

norealroots opened the pull request:
- Updates the the README for the workshop
- Added the Apache License 2.0 to the workshop content
- Gave proper attribution the the Al Sweigart, the author of the book the workshop content is based on.

norealroots committed:
Update README.md

> Added a line to direct a viewer towards the releases section
> Added the "Overview" section from `compwshop_summary.pdf` to give a more obvious credit to original sources

norealroots committed:
Create LICENSE.md

> Added a License to this work and gave proper attribution to the Al Sweigart, the author of the book this work is based upon.

tofu-rocketry commented on LICENSE.md:
```diff
@@ -0,0 +1,27 @@
+
+# Invent Your Own Computer Games with Python 3rd Edition
+
+This work is based on the book “Invent Your Own Computer Games with Python 3rd Edition” by Al Sweigart (ISBN-10 0982106017), and is licensed under Creative Commons Attribution-Noncommercial-Share Alike 3.0 United States License.
```
```suggestion
This work is based on the book “Invent Your Own Computer Games with Python 3rd Edition” by Al Sweigart (ISBN-10 0982106017) which is licensed under a Creative Commons Attribution-Noncommercial-Share Alike 3.0 United States License.
```

It was ambiguous as to whether you were referring to this work or the book.

norealroots replied:
Agreed, this is a better structure.

tofu-rocketry commented on LICENSE.md:
````diff
@@ -0,0 +1,27 @@
+Copyright 2019 Science and Technology Facilities Council, U.K. Research and Innovation
+
+```
+   Licensed under the Apache License, Version 2.0 (the "License");
````
I do not think you can release this under an Apache licence if it's based on a CC BY-NC-SA licenced work:

>### Versions 2.0, 2.5, and 3.0
> 
> Your contributions to adaptations of BY-NC-SA 2.0, 2.5 or 3.0 materials may only be licensed under:
> 
>    - The license used for the original work, or a later version of that BY-NC-SA license.
>    - Ported versions of that BY-NC-SA license, the same or later version as the licensed work.

norealroots replied:
I hadn't realised that, I'll move the License to a CC one.

tofu-rocketry replied:
I forgot the link for my quote for reference: https://creativecommons.org/share-your-work/licensing-considerations/compatible-licenses

norealroots replied:
Thanks - I had found this to check what needed doing. I have changed the workshop's license to a `BY-NC-SA 4.0` license, which meets the criteria of `The license used for the original work, or a later version of that BY-NC-SA license` for a 3.0 license.

tofu-rocketry reviewed (changes requested):
You need to check the licensing.

norealroots committed:
Update LICENSE.md

Changed license to a Creative Commons Attribution-NonCommercial-ShareAlike 4.0 International License, as otherwise it would have clashed with the License on the Invent Your Own Computer Games with Python book.

norealroots committed:
Update LICENSE.md

Accepted suggested sentence structure changes.

Co-Authored-By: Adrian Coveney <tofu-rocketry@users.noreply.github.com>

jrha reviewed (approved):
LGTM!
